src/CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ if(HAVE_BFD_DISASM)
   set(BFD_DISASM_SRC bfd-disasm.cpp)
 endif()
 
-add_executable(bpftrace
+if(BUILD_FUZZ)
+  set(MAIN_SRC fuzz_main.cpp)
+  set(BPFTRACE bpftrace_fuzz)
+else()
+  set(MAIN_SRC main.cpp)
+  set(BPFTRACE bpftrace)
```

Comment:
nit: might be good to unset (`set(MAIN_SRC "") iirc`) these at the end to avoid them leaking into cmake state.